win32ss/user/ntuser/menu.c

```diff
@@ -4160,6 +4161,8 @@ static INT FASTCALL MENU_TrackMenu(PMENU pmenu, UINT wFlags, INT x, INT y,
             /* Find a menu for this mouse event */
             pmMouse = MENU_PtMenu( mt.TopMenu, mt.Pt );
 
+            PWND pWnd = ValidateHwndNoErr(mt.TopMenu->hWnd);
```

Comment:
Please move this to where this variable is actually used (around line 4200)
Also, having this variable declaration in the middle of code is not valid in C.

Reply:
Moved variable declaration to the top of this function. Removed statement here and placed it near line 4182 and 4203 as requested. Thanks.